src/EPR.RegulatorService.Facade.Core/Clients/PrnBackendServiceClient/IPrnBackendServiceClient.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace EPR.RegulatorService.Facade.Core.Clients.PrnBackendServiceClient;
+public interface IPrnBackendServiceClient
```

Comment:
Minor, but we could add a line break here:
```suggestion
﻿namespace EPR.RegulatorService.Facade.Core.Clients.PrnBackendServiceClient;
﻿
public interface IPrnBackendServiceClient
```